exasol_integration_test_docker_environment/cli/options/test_environment_options.py

```diff
@@ -42,6 +42,12 @@
                  help="""Database port of external Exasol DB, needs to be set if --environment-type=external_db"""),
     click.option('--external-exasol-bucketfs-port', type=int, default=6583,
                  help="""Bucketfs port of external Exasol DB, needs to be set if --environment-type=external_db"""),
+    # should parameter be optional?
+    # this may correspond to potential exception to be raised in
+    # lib/test_environment/spawn_test_environment.py
+    # _create_external_db_environment
+    click.option('--external-exasol-ssh-port', type=int, default=22,
```

Comment:
good question, to be honest, don't know, yet

Reply:
We will not define a default value here as the value depends on the version of the Exasol database, 
e.g. version 7.1 the default port is 22 for version 8.0 it is 20002.